### PR TITLE
Update Vue Post

### PR DESCRIPTION
**Pull request by rdegges**
Making a lot of improvements to an old Go + Vue post.

### CODE REVIEW

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md [hunk 1]
```diff
@@ -49,15 +49,23 @@ With the GOPATH defined, you now can create a directory for your project:
 mkdir -p $GOPATH/src/github.com/{YOUR_GITHUB_USERNAME}/kudo-oos
 ```
 
-To get your SPA off the ground quickly, leverage the scaffolding functionality from [vue-cli](https://cli.vuejs.org/). The CLI will prompt you with a series of options - pick the technology appropriate for this project: `vue.js`, `vuex`, and, `webpack`.
+To get your SPA off the ground quickly, leverage the scaffolding functionality of [vue-cli](https://cli.vuejs.org/). The CLI will prompt you with a series of options - pick the technology appropriate for this project: `vue.js`, `vuex`, and, `webpack`.
```
Comment (mraible) [inline _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: leverage > you can leverage

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md [hunk 2]
```diff
@@ -128,15 +135,33 @@ Next, install the Okta Vue SDK by running the following command:
...
+
+## Install Your Vue App Dependencies
+
+You'll also need to install a few dependencies in order to get your project
+running. Run the following command to install them all:
+
+```bash
+yarn add vue-router
+yarn add vuex
+yarn add axios
```
Comment (mraible) [inline _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: Put all these one one line and version them:

```
yarn add vue-router vuex axios
```

Without versions, you'll be screwed when any of them does a major release and breaks backward compatibility.

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md [hunk 3]
```diff
@@ -150,8 +175,8 @@ import GitHubRepoDetails from './components/GithubRepoDetails';
 
 Vue.use(VueRouter);
 Vue.use(Auth, {
-  issuer: {ADD_YOUR_DOMAIN},
-  client_id: {ADD_YOUR_CLIENT_ID},
+  issuer: '{OKTA_ORG_URL}/oauth2/default',
```
Comment (mraible) [inline _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]:
```suggestion
  issuer: '{yourOktaDomain}/oauth2/default',
```

Change to yourOktaDomain if you want it pre-populated.

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md [hunk 4]
```diff
@@ -669,177 +763,180 @@ Note that we are calling `router.beforeEach(Vue.prototype.$auth.authRedirectGuar
...
+        coll := session.DB("").C(collectionName)
+
+        for _, kudo := range kudos {
+                _, err := coll.Upsert(bson.M{"repoId": kudo.RepoID, "userId": kudo.UserID}, kudo)
+                if err != nil {
+                        return err
+                }
+        }
+
+        return nil
```
Comment (mraible) [inline _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: Are you sure all these whitespace changes are correct?

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md [hunk 5]
```diff
@@ -1151,9 +1260,63 @@ func main() {
...
+
+{% raw %}
+```go
+package main
+
+import (
+        "log"
+        "os"
+        "github.com/globalsign/mgo"
+)
```
Comment (mraible) [inline _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: Formatting looks funny, like tabs. Is that just a Go thing?

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md (hunk 4 above)
Comment (rdegges) [reply _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: Yes! =)

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md (hunk 5 above)
Comment (rdegges) [reply _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: Yes -- Go uses tabs (not spaces).

File: _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md (hunk 2 above)
Comment (rdegges) [reply _source/_posts/2018-10-23-build-a-single-page-app-with-go-and-vue.md]: Thanks! Resolved =)

**Comment by rdegges** [comment]
@mraible can you approve? Made some changes =)

**Comment by mraible** [comment]
I just noticed you didn't include a changelog. Not sure it's necessary, but we should add them when we update a post. Here's a template I use:

```
**Changelog:**

* MMM DD, YYY: Brief summary of changes. See the example app changes in [example-app-name#pull-request-number](https://github.com/oktadeveloper/example/pull/#); changes to this post can be viewed in [okta.github.io#pull-request-number](https://github.com/okta/okta.github.io/pull/#).
```

**Pull request merged**